src/lib/sqlite-versions.ts

```diff
@@ -96,3 +123,52 @@ export async function removeLegacySqliteIntegrationPlugin( installPath: string )
 		Sentry.captureException( error );
 	}
 }
+
+/**
+ * Updates the SQLite integration in a site if it's outdated compared to the version
+ * located in the server files.
+ *
+ * If the SQLite integration is not installed, it will be installed if the site
+ * doesn't provide the configuration file `wp-config.php`.
+ *
+ * @param sitePath Path of the site.
+ */
+export async function keepSqliteIntegrationUpdated( sitePath: string ) {
+	const sqlitePath = path.join( sitePath, 'wp-content', 'mu-plugins', SQLITE_FILENAME );
+	const hasWpConfig = await fs.pathExists( path.join( sitePath, 'wp-config.php' ) );
+	const sqliteInstalled = await isSqlLiteInstalled( sqlitePath );
+	const sqliteOutdated = sqliteInstalled && ( await isSqliteInstallationOutdated( sqlitePath ) );
+
+	if ( ( ! sqliteInstalled && ! hasWpConfig ) || sqliteOutdated ) {
+		await installSqliteIntegration( sitePath );
+	}
+}
+
+/**
+ * Installs the SQLite integration in a site. This includes the must-used plugin
+ * and the database file.
+ *
+ * @param sitePath Path of the site.
+ */
+export async function installSqliteIntegration( sitePath: string ) {
+	await downloadSqliteIntegrationPlugin();
+	const wpContentPath = path.join( sitePath, 'wp-content' );
+	const databasePath = path.join( wpContentPath, 'database' );
+
+	await fs.mkdir( databasePath, { recursive: true } );
+
+	const dbPhpPath = path.join( wpContentPath, 'db.php' );
+	await fs.copyFile( path.join( getServerFilesPath(), SQLITE_FILENAME, 'db.copy' ), dbPhpPath );
+	const dbCopyContent = ( await fs.readFile( dbPhpPath ) ).toString();
```

Comment:
What do you think about using `await fs.readFile( dbPhpPath, 'utf8' );` ?
I think it will be more consistent with other parts of the codebase.

Reply:
Good catch. I didn't notice this when moving the code ([ref](https://github.com/Automattic/studio/pull/422/files#diff-4897e47ea3fb6cdbb1798e8dce993fa538a1d6d56b425eabbf82d945889ec508L146)). I've addressed in https://github.com/Automattic/studio/pull/422/commits/47d216b8fcbcaead3a563bd15f1a22efd5319d77.